contracts/shared/Types.sol

```diff
@@ -18,6 +18,14 @@ struct AftermarketDeviceOwnerPair {
     address owner;
 }
 
+struct MintSyntheticDeviceBatchInput {
```

Comment:
Note to me: the difference between this and the old input is that we don't require an owner signature here.